src/components/coffeechat.ts

```diff
@@ -0,0 +1,392 @@
+import { Client } from 'discord.js-commando';
+import { Database } from 'sqlite';
+import { openDB } from './db';
+import _ from 'lodash';
+import { Person, stableMarriage } from 'stable-marriage';
+
+const COFFEE_ROLE_ID: string = process.env.COFFEE_ROLE_ID || '.';
+const TARGET_GUILD_ID: string = process.env.TARGET_GUILD_ID || '.';
+const RANDOM_ITERATIONS = 1000;
+
+interface historic_match {
+  first_user_id: string;
+  second_user_id: string;
+  TIMESTAMP: string;
```

Comment:
`match_date` maybe?

Reply:
also can we use camel case for `historic_match`? that's the convention we use, only DB stuff is snake case i think